sync/src/main/java/tech/pegasys/artemis/sync/SyncManager.java

```diff
@@ -130,16 +130,18 @@ synchronized boolean isSyncQueued() {
 
   public SyncingStatus getSyncStatus() {
     final boolean isSyncActive = isSyncActive();
-    if (!isSyncActive) {
-      final SyncStatus syncStatus =
-          new SyncStatus(UnsignedLong.ZERO, UnsignedLong.ZERO, UnsignedLong.ZERO);
-      return new SyncingStatus(false, syncStatus);
-    } else {
-      final UnsignedLong highestSlot = findBestSyncPeer().get().getStatus().getHeadSlot();
-      final SyncStatus syncStatus =
-          new SyncStatus(peerSync.getStartingSlot(), storageClient.getBestSlot(), highestSlot);
-      return new SyncingStatus(isSyncActive(), syncStatus);
+    if (isSyncActive) {
+      Optional<Eth2Peer> bestPeer = findBestSyncPeer();
+      if (bestPeer.isPresent()) {
+        UnsignedLong highestSlot = bestPeer.get().getStatus().getHeadSlot();
+        final SyncStatus syncStatus =
+            new SyncStatus(peerSync.getStartingSlot(), storageClient.getBestSlot(), highestSlot);
+        return new SyncingStatus(isSyncActive(), syncStatus);
```

Comment:
use the boolean rather than the function here for isSyncActive?

Reply:
yep